doc-content/enterprise-only/processes/process-instance-filtering-proc.adoc

```diff
@@ -3,60 +3,36 @@
 For process instances in *Menu* -> *Manage* -> *Process Instances*, you can use the *Filters* and *Advanced Filters* panels to sort process instances as needed.
 
-.Filtering Process Instances - Default View
-image::processes/ProcessInstancesSearchSmall.png[Filtering Process Instances - Default View]
-
-You can filter process instances by the following attributes in the *Filters* panel:
-
-*Id*:: Filter by process instance ID.
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Manage* -> *Process Instances*.
+. On the *Manage Process Instances* page, click the filter icon on the left of the page to expand the *Filters* pane and select the filters you want to use:
++
+* *State*: Filter process instances based on their state (*Active*, *Aborted*, *Completed*, *Pending*, and *Suspended*).
+* *Errors*: Filter process instances that contain at least one or no errors.
+* *Filter By*: Filter process instances based on the following attributes:
+** *Id*: Filter by process instance ID.
 +
 Input: `Numeric`
 
-*Initiator*:: Filter by the user ID of the process instance initiator.
+** *Initiator*: Filter by the user ID of the process instance initiator.
 +
 The user ID is a unique value, and depends on the ID management system.
 +
 Input: `String`
 
-*Correlation key*:: Filter by correlation key.
+** *Correlation key*: Filter by correlation key.
 +
 Input: `String`
 
-*Description*:: Filter by process instance description.
+** *Description*: Filter by process instance description.
 +
 Input: `String`
-
-State::
-Filter by different states. You can select more than one status to display results that meet any of the selected states. Removing the status filter displays all processes, regardless of status.
-+
-The following filter states are available:
-+
-** Active
-** Aborted
-** Completed
-** Pending
-** Suspended
-
-Errors::
-Filter by process instances with or without errors.
-
-Name:: Filter by project name.
-
-Date and time filtering::
-+
-Start Date::: When the process was started.
-Last Update::: When the process was updated or modified.
-
-:: Each of these filters have the following quick filter options:
-
-** Last Hour
-** Today
-** Last 24 Hours
-** Last 7 Days
-** Last 30 Days
-** Custom
-+
-Selecting *Custom* date and time filtering opens a calendar tool for selecting a date and time range.
-+
-.Process Instances Search by Date Range
-image::processes/DateRangeSearch.png[Search by Date Range]
+* *Name*: Filter process instances based on process definition name.
+* *Definition ID*: The ID of the instance definition.
+* *Deployment ID*: The ID of the instance deployment.
+* *SLA Compliance*: SLA compliance status (*Aborted*, *Met*, *N/A*, *Pending*, and *Violated*).
+* *Parent Process ID*: The ID of the parent process.
+* *Start Date*: Filter process instances based on their creation date.
+* *Last update*: Filter process instances based on their last modified date.
+
+You can also use the *Advanced Filters* option to create custom filters in {CENTRAL}.
```

Comment:
Hey @emmurphy1, I guess, the rendered output link is not reflecting the changes. Can you please attach the updated links?

Reply:
The links are correct. Check section 4.1.

Reply:
@emmurphy1, Apologies for the same. I was checking different rendered output link.